Resync `css-tables` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 190e2361e15156319d5b9104a9d7f6d0c63bc977
<pre>
Resync `css-tables` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=303525">https://bugs.webkit.org/show_bug.cgi?id=303525</a>

Reviewed by Brandon Stewart.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/cb0485e0505240c7fa264b6d0e0f78165432674b">https://github.com/web-platform-tests/wpt/commit/cb0485e0505240c7fa264b6d0e0f78165432674b</a>

* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-relative-positioning.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/transition-table-row-group-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/fixed-layout-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/fixed-layout-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-collapsed-row-or-column-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/303917@main">https://commits.webkit.org/303917@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/302a3300c18ffc18ee27dafd50207c2c4149ac86

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/133926 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/6437 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/45122 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/141506 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/85987 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/304ce841-0623-43f9-b694-e929fc8cbe89) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/6968 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/6301 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/102444 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/69751 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/67b1a7ed-e23a-419a-8469-4bb5e9525cde) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/136873 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/4897 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/120057 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/83243 "Passed tests") | | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/89f7db66-7acb-4222-97f5-6917f5ae7a04) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/4775 "Passed tests") | [✅ 🧪 api-mac-debug](https://ews-build.webkit.org/#/builders/165/builds/2394 "Passed tests") | [⏳ 🛠 wpe-cairo ](https://ews-build.webkit.org/#/builders/WPE-Cairo-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/113936 "Found 1 new API test failure: TestWebKitAPI.IndexedDB.IndexedDBSuspendImminently (failure)") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/38177 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/144150 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/6107 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/38756 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/110810 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/6189 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/5185 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/111019 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/28167 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/4630 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/116315 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/59854 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/6159 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/34575 "Passed tests") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/6005 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/6250 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/6113 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

stwrt reviewed (approved):
LGTM if tests are passing.

Ahmad-S792 committed:
Resync `css-tables` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=303525

Reviewed by Brandon Stewart.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/cb0485e0505240c7fa264b6d0e0f78165432674b

* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-relative-positioning.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/transition-table-row-group-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/fixed-layout-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/fixed-layout-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-collapsed-row-or-column-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/w3c-import.log:

Canonical link: https://commits.webkit.org/303917@main
